tests/i18n/tests.py

```diff
@@ -68,8 +68,10 @@ def test_plural(self):
         """
         self.assertEqual(ungettext("%d year", "%d years", 0) % 0, "0 année")
         self.assertEqual(ungettext("%d year", "%d years", 2) % 2, "2 années")
-        self.assertEqual(ungettext("%(size)d byte", "%(size)d bytes", 0) % {'size': 0}, "0 octet")
-        self.assertEqual(ungettext("%(size)d byte", "%(size)d bytes", 2) % {'size': 2}, "2 octets")
+        self.assertEqual(ungettext("%(size)d byte", "%(size)d bytes", 0) % {
```

Comment:
Could you please revert the unrelated line length changes? I pushed some fixes to your branch before I realized that this file is also changed. Maybe your IDE went wild? We allow longer lines per https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/.